website/docs/r/vapp_vm.html.markdown

```diff
@@ -197,8 +216,8 @@ The following arguments are supported:
 * `vapp_name` - (Required) The vApp this VM belongs to.
 * `name` - (Required) A name for the VM, unique within the vApp 
 * `computer_name` - (Optional; *v2.5+*) Computer name to assign to this virtual machine. 
-* `catalog_name` - (Required) The catalog name in which to find the given vApp Template
-* `template_name` - (Required) The name of the vApp Template to use
+* `catalog_name` - (Optional; *v2.8+*) The catalog name in which to find the given vApp Template
```

Comment:
2.8 should be repaced to 2.9 across this PR.

Reply:
updated